NMS-15514: Upgrade to Newts 3.0.0 (#5939)
OpenNMS/opennms: docs/modules/deployment/pages/time-series-storage/newts/configuration.adoc

|===

> | Name  | Description   | Default

Reviewer: ```suggestion
| Name  | Description   | Default Value
```

> See http://docs.datastax.com/en/cassandra/2.1/cassandra/dml/dml_config_consistency_c.html[Configuring Data Consistency] in the Cassandra documentation for a list of available options.

Reviewer: ```suggestion
See http://docs.datastax.com/en/cassandra/2.1/cassandra/dml/dml_config_consistency_c.html[Configuring Data Consistency] in the official Cassandra documentation for a list of available options.
```

| ANY

| org.opennms.newts.config.driver_settings_file

> | Cassandra driver settings file - overwrites all other driver related settings

Reviewer: ```suggestion
| Cassandra driver settings file.
The configuration settings defined in this file overwrite all other driver-related settings.
```

> See https://docs.datastax.com/en/developer/java-driver/4.14/manual/core/configuration/reference/[Reference configuration] for details.

Reviewer: ```suggestion
See https://docs.datastax.com/en/developer/java-driver/4.14/manual/core/configuration/reference/[Reference Configuration] in the official DataStax documentation for details.
```
